README.md

```diff
@@ -17,13 +17,13 @@ npm install --save-dev eslint-plugin-mocha
 
 ### Configuration via `eslint.config.js`
 
-To use this plugin with [the new eslint configuration format (flat config)](https://eslint.org/docs/latest/use/configure/configuration-files-new):
+To use this plugin with [eslint flat configuration format](https://eslint.org/docs/latest/use/configure/configuration-files-new):
 
 ```js
 import mochaPlugin from "eslint-plugin-mocha";
 
 export default [
-    mochaPlugin.configs.flat.recommended, // or `mochaPlugin.configs.flat.all` to enable all
+    mochaPlugin.configs.recommended, // or `mochaPlugin.configs.all` to enable all
```

Comment:
It's a bit confusing that the config structure was changed in a TS migration PR and that the breaking change wasn't listed in the 11.0.0 release notes.

It's easy enough to fix but updating the release notes would make it easier for people to til what happened when their dependency updates fail.

Reply:
Thanks. I’ve added a note the the release notes.